Python/Product/PythonTools/PythonTools/Debugger/DebugAdapter/DebugAdapterAttachProcess.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PythonTools.Debugger.DebugAdapter;
+using Microsoft.PythonTools.Infrastructure;
+using Microsoft.PythonTools.Logging;
+using Microsoft.VisualStudio.Debugger.DebugAdapterHost.Interfaces;
+using Microsoft.VisualStudio.Debugger.Interop;
+
+namespace Microsoft.PythonTools.Debugger {
+    internal sealed class DebugAdapterAttachProcess : ITargetHostProcess, IDisposable {
+        private const int _debuggerConnectionTimeout = 60000; // 60 seconds. Can take a long time to connect
+        private DebugAdapterAttachProcessStream _stream;
```

Comment:
should we set `_stream` to `null` here?

Reply:
Default of any reference type should be null:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values